HHS/Head-Start-TTADP
# [TTAHUB-1638] Remove orphan files and resources

## Description of change

This change cleans orphan files and resources from their respective tables. It also removes the orphan files from S3.


## How to test

- Review all code make sure all tests pass.

1. Create a AR and Goal with a resource and file.
2. Remove the goal and save.

![image](https://user-images.githubusercontent.com/84350609/234041524-ee4e60f5-4e23-4b28-9145-ec1b0edfc27b.png)

4. Ensure both the resource and goal have been removed from their tables.
5. Check S3 to make sure the file has been removed.

**Note:** Removing individual resources or file from an **AR** will NOT remove the from the respective table. This is because the ObjectiveResource and ObjectiveFile still exists. However removing individual files and resources from the RTR does remove from the corresponding table.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1638


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
